ansible/pygrid_deploy.yaml

```diff
@@ -0,0 +1,90 @@
+- hosts: hosts
```

Comment:
you might want to start (first line) the yaml file with `---`
also, you should also include a `name` on your playbook